docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -145,6 +146,13 @@ $ htpasswd -nbs internal <my-secret-password>
 
 By default, Pelorus gathers the data from the Prometheus instance deployed in the same cluster in which it is running. To collect data across multiple OpenShift clusters, additional Prometheus scrape hosts have to be configured. To do this `federated_prometheus_hosts` and `external_prometheus_hosts` configuration options are used.
 
+###### federate_openshift_monitoring
```

Comment:
update docs (its object instead of boolean)

an example how to use metrics_filter field would be interesting